packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1622,15 +1629,18 @@ export class TransactionController extends BaseControllerV1<
     ) as TransactionParams;
 
     const updatedTransaction = merge(transactionMeta, editableParams);
+    const ethQuery = this.#multichainTrackingHelper.getEthQuery({
+      networkClientId: transactionMeta.networkClientId,
+      chainId: transactionMeta.chainId,
+    });
     const { type } = await determineTransactionType(
       updatedTransaction.txParams,
-      this.#multichainTrackingHelper.getEthQuery({
-        networkClientId: transactionMeta.networkClientId,
-        chainId: transactionMeta.chainId,
-      }),
+      ethQuery,
     );
     updatedTransaction.type = type;
 
+    await this.#updateTransactionLayer1GasFee(ethQuery, updatedTransaction);
```

Comment:
Intentionally skipping to test this case since we don't have any `Layer1GasFeeFlow` right now.